traffic_portal/app/src/common/modules/form/deliveryService/form.deliveryService.anyMap.tpl.html

```diff
@@ -156,7 +156,7 @@ <h3>Current Value</h3>
                         </div>
                         </label>
                         <div class="col-md-10 col-sm-10 col-xs-12">
-                            <select id="cdn" name="cdn" class="form-control" ng-model="deliveryService.cdnId" ng-options="cdn.id as cdn.name for cdn in cdns" required ng-disabled="!settings.isNew">
+                            <select id="cdn" name="cdn" class="form-control" ng-model="deliveryService.cdnId" ng-options="cdn.id as cdn.name for cdn in cdns" required>
```

Comment:
do u want to disable this one?

Reply:
Nope, since SSLkeys are part of only 3 services: HTTP, DNS and Steering